sampleinput.yaml

```diff
@@ -0,0 +1,14 @@
+creds:
```

Comment:
I had given it a thought but same as my comment on the next review, at that time it was easier to parse the other way considering the current code structure. I was focusing on just getting it to work first. but I like this suggestion. I will try changing the format to this.